src/main/java/com/exercise/tasksmanagementsystem/entity/Task.java

```diff
@@ -1,6 +1,15 @@
 package com.exercise.tasksmanagementsystem.entity;
 
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+
+@Entity
```

Comment:
it would be a table in your database, so you have also add @Table annotation with a table name, we will create it next